src/bootstrap.styles.loader.js

```diff
@@ -0,0 +1,60 @@
+/* eslint func-names: 0 */
+
+import path from 'path';
+
+import processModules from './utils/processModules';
+import getFontsPath from './utils/getFontsPath';
+import createUserImport from './utils/createUserImport';
+import createBootstrapImport from './utils/createBootstrapImport';
+
+module.exports = function() {
+  if (this.cacheable) this.cacheable();
+
+  const config = global.__BOOTSTRAP_CONFIG__;
+  const bootstrapVersion = parseInt(config.bootstrapVersion, 10);
+  const {
+    styles,
+    bootstrapPath,
+    useFlexbox,
+    preBootstrapCustomizations,
+    bootstrapCustomizations,
+  } = config;
+
+  const bootstrapRelPath = path.relative(this.context, bootstrapPath);
+
+  const processedStyles = [];
+
+  if (bootstrapVersion === 4 && useFlexbox) {
+    processedStyles.push('$enable-flex: true;');
+  }
+
+  if (preBootstrapCustomizations) {
+    processedStyles.push(
+      createUserImport(preBootstrapCustomizations, this)
+    );
+  }
+
+  processedStyles.push(
+    createBootstrapImport('variables', bootstrapVersion, bootstrapRelPath)
+  );
+
+  if (bootstrapVersion === 3) {
+    processedStyles.push(
+      `$icon-font-path: "${getFontsPath(bootstrapRelPath, this)}";`
+    );
+  }
+
+  if (bootstrapCustomizations) {
+    processedStyles.push(
+      createUserImport(bootstrapCustomizations, this)
```

Comment:
It's not yet added, because I'm figuring out strategy for it.

Here is my conclusion.

There are 2 conventional use cases of loading CSS:

1. Common. When user provides one endpoint (SCSS file) where all CSS is collected via `import`s. This can be handled by `bootstrap-loader` as it uses same loaders for loading user's & bootstrap's css.
1. CSS Modules. There is no endpoint here, all CSS is loaded from JS components, so in this case `bootstrap-loader` will load Bootstrap and pass application endpoint to the next loaders (without touching them), where CSS Modules will be handled (on webpack config level, `config.module.loaders` section). We will provide example of it. So if user uses CSS Modules and will provide SCSS endpoint here (to some common CSS, which is not used as CSS Modules) — it will be loaded globally, just like Bootstrap styles.